src/omv/imlib/framebuffer.c

```diff
@@ -381,6 +381,19 @@ void framebuffer_free_current_buffer()
     }
 }
 
+void framebuffer_setup_buffers()
+{
+    #ifdef __DCACHE_PRESENT
+    for (int32_t i = 0; i < framebuffer->n_buffers; i++) {
+        if (i != framebuffer->head) {
+            vbuffer_t *buffer = framebuffer_get_buffer(i);
+            // Make sure all cached CPU writes are flushed before returning the buffer.
```

Comment:
This comment (which is copied from `framebuffer_free_current_buffer`) is wrong, because this invalidates the cache not flush writes.

Reply:
It's job is to the same as the other line except for all other buffers versus the one at framebuffer->head. So, the comment is correct.

Both comments are wrong using the word flushed otherwise.

Reply:
Updated the wording.